userauth_test.go

```diff
@@ -129,7 +129,7 @@ func TestUserTokenAuthenticator_ValidateSignature(t *testing.T) {
 	// Set up request defaults
 	r, _ := http.NewRequest("", "/", nil)
 	r.Host = "localhost"
-	r.Header.Set("X-Amz-Security-Token", "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL2R1bW15LmVnYS5uYmlzLnNlIiwic3ViIjoiZHVtbXkiLCJleHAiOjE3MTk2NzY5ODB9.oa5Lx9MBe_Fv6sBAg3YnsWau9VQLAcnb-6G_0sNx6857qMSeUKnv4bNE7gZ1FGR60ofoPJZ4Pg2Ni2VyYQ1AwP4LNeCGiQ5vAmdNKwt6QrBrsxAeSK3G4OXRD8uoK9t1iK6Gkk0GMZgjnJF9a5YkhYYYMKY6AcXP3ObV5bufb9s86rcZG_bZ__dudH6QbSCVvDnJrzrAK_5Wzoz8DNwTMDZhCGar4_sraj3_5TeXIUHu0zZGg-4ni9VDQ-EXN7ZJbMKmwB_j9EYLhQGA4oz8BbD3y1h9xkrDkqqZWHYcxW1Kn3ZPGfb-bLlTIjzrLOO7MkSZid4p6cKdcpQ_LV5qALRr-L-GMF4RP8nn47_8mw1olXIFSlr-dp8yladploFIsEiol3Hwjvan9hq8QTWRaOj-SAmGqGnkJ3WkS7TTSlRP3IOXho_UVbRtBfbeE9k17aFjdHZQSd54DtWFQL_ot8khyrJGsNKtsyLdVkE1fflrF3G21Mn_H9LHnieNhahx_zRnCS8z7v1L1x2Xe2h8XHyu-Bu9el2weYcMGIUqDWJAs1i5o0XAN5wRr7q7XVBlCpS6vwnmDvYTdGAW2EwpSLkDcwq5ONPtPF8qF95RYPaGaOHeejN7qNIj-aL2pYfr_qLMFuLYV3LH1J1jhqmcvukMmJNyC72h1s872eeGX0Q")
+	r.Header.Set("X-Amz-Security-Token", "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL2R1bW15LmVnYS5uYmlzLnNlIiwic3ViIjoiZHVtbXkiLCJleHAiOjE3MTk2NzY5ODB9.XjKAkrd8OCV_ApFiMwEIZiYM4a2QFCr4Ik9A3AqPNtV1Mt2IwyMZHrGGuljkf6YT-ZDsk6ONrXxoL5Vifs7px8BULpcoscWtqmW2TM6tKMe0notaXC8kuPMnGK5VEKul1ayibOK0n2QKoOgHzALELQRC44_oOylUoZaQ9OYpuvF8nBf-dn_lSlxsarzDh9iDI95BiXRW9ARvUGtaLESsS-Oar7Z36P8QM3IV-Omiorv8hKrlg1rjCywF5MlWDQSevLiW9Gj_vwBnyKGX06yYlk8J0vJ_yxh-YFZvhzR149MM-3nvHo8CL3LeO4AVRalvRcFnpZgBu60BlZJ7IGIyAA")
```

Comment:
can we generate these on the fly or set the expiry 10 years in the future to prevent the issue with an expired token?